roles/common/tasks/ntp.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- name: 'NTP. Install Package'
+  yum:
+    name: ntp
+    state: present
+    update_cache: true
+
+- name: 'NTP. Configure Service'
+  template:
+    src: ntp.conf
+    dest: /etc/ntp.conf
+    owner: root
+    group: root
+    mode: 0644
+    backup: true
+  notify:
+    - restart ntp
+
+- name: 'NTP. Start Service'
+  service:
+    name: ntpd
+    state: started
+    enabled: true
+
+- name: 'Timezone. Set to Europe/Kiev'
+  timezone:
+    name: Europe/Kiev
```

Comment:
Please also parametrize this.

Reply:
Migration to jinja + fixing template source file